src/parser.h

```diff
@@ -27,6 +29,8 @@ class XASMParser {
     Lexer lexer;
     Token currentToken;
     Token nextToken;
+    std::map<std::string, uint> labels;
+    uint pc;
```

Comment:
Please use `u32` for this instead of `uint` for future consistency.

Reply:
Used it to not create another dependency (include in header). But i'll go for u16 instead. Should be enough for pc. What do you think?